.readthedocs.yml

```diff
@@ -1,20 +1,23 @@
-# readthedocs.yml
-# Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
-
 version: 2
 
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  jobs:
+    post_checkout:
+      - git fetch --unshallow || true
+    pre_install:
+      - git update-index --assume-unchanged docs/conf.py
```

Comment:
What's modifying conf.py?

Reply:
I thought that was some conda file? I dont remeber.

Reply:
I thought it changes the conda file and the docs/conf.py but since we lack a conda file here, we don't need to avoid it.

I can remove this?